pkg/aggregator/demultiplexer.go

```diff
@@ -835,6 +857,20 @@ func (d *ServerlessDemultiplexer) ForceFlushToSerializer(start time.Time, waitFo
 	d.flushLock.Lock()
 	defer d.flushLock.Unlock()
 
+	// only used when we're using flush/serialize in parallel feature
+	logPayloads := config.Datadog.GetBool("log_payloads")
```

Comment:
Would it make sense to move this into the `startSendingIterableSeries`? Same code with same config var is used when calling it from the demultiplexer.

Reply:
`logPayloads` is also used later in this function: https://github.com/DataDog/datadog-agent/pull/11113/files/775cf7b897e3fcf8e990071877d953b77147bd99#diff-ad4da2ba7c34444c3d1583595302c3b00123265986f7ae6e6c855045d6e0f8e3L579.